testdata/prometheus/prometheus.yml

```diff
@@ -34,3 +34,105 @@ scrape_configs:
     - 127.0.0.1:7773
     labels:
       instance: pmm-server
+- job_name: mysqld_exporter_agent_id_4ab6cc14-10b8-4bcc-bba2-aab95e81299c_hr
```

Comment:
It seems this all stuff should be reverted??

Reply:
Yes, it should be reverted